src/shared/components/GenericList/GenericList.js

```diff
@@ -217,6 +217,8 @@ export const GenericList = ({
       title={title}
       headerActions={!headerActionsEmpty && headerActions}
       data-testid={testid}
+      disableMargin
+      className="bsl-margin-top--md"
```

Comment:
Please import spacing from UI5 and make it like this

```suggestion
      style={spacing.sapUiMediumMarginTop}
```